pkg/console/console.go

```diff
@@ -53,14 +55,39 @@ func (s StandardConsole) Run(cmd string, opts ...func(cmd *exec.Cmd)) (string, e
 	for _, o := range opts {
 		o(c)
 	}
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	// Start the timer log
+	go displayProgress(ctx, s.logger, 10*time.Second, fmt.Sprintf("Still running command '%s'", cmd))
+
+	// Run the command
 	out, err := c.CombinedOutput()
+
+	// Stop the timer log
+	cancel()
```

Comment:
isn't this redundant? we do already call `defer cancel()`

Reply:
umm that its true, maybe I was testing this manually or something?